x/evm/types/testutils/types.go

```diff
@@ -228,7 +223,7 @@ func RandomToken() types.ERC20TokenMetadata {
 		TokenAddress: RandomAddress(),
 		TxHash:       RandomHash(),
 		Status:       1 << rand.I64Between(0, int64(len(types.Status_name))),
-		MinAmount:    sdk.NewInt(rand.PosI64()),
+		MinAmount:    sdk.ZeroInt(),
```

Comment:
Fair point. I just tried it, but looks like it breaks import/export because the field is still present, the default value `sdk.Int{nil}` when exported and then imported returns `sdk.Int{sdk.BigInt{false, nil}}` on unmarshaling, so they aren't the same. We can't remove this field, so shall I set it to the `0` value in it's constructor for import/export consistency?

Reply:
It's not used anymore. It seems that protobuf will ignore the reserved field when unmarshalling old ERC20 metadata. I'll remove it.

Reply:
removal is always backward-compatible with protobuf. reserving makes it better by making sure the same field number isnt gonna be reused in the future.

Reply:
Cool, removed it.